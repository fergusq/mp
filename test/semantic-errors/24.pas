program p;
procedure f(r:real);
begin writeln(1)end;
begin f()end.
