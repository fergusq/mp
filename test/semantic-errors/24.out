Semantic error: Wrong number of arguments given for f
